src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -12487,3 +12551,107 @@ pbsv1mod_meth_release_nodes(PyObject *self, PyObject *args, PyObject *kwds)
 	hook_set_mode = hook_set_mode_orig;
 	return (py_return);
 }
+
+
+/**
+ *
+ * @brief
+ *	Returns a Python List of _server_attribute objects or  or NULL on error.
+ * @param[in]	phead	- pointer to the head of the list containing data.
+ *
+ * @return 	PyObject *
+ * @retval	<object>	- the Python list object holding
+ * @retval				the _server_attribute objects.
+ * @retval	NULL		- if an error occurred.
```

Comment:
Please put an @note here that the returned <object> contains a new reference, which must by Py_CLEAR-ed later. This is to guard its future use.

Reply:
Note was added.